### DISCUSSION TITLE
安装完成后报 500 错误

### DISCUSSION THREAD

**Discussion by Genius-liu** [spencerwooo/onedrive-vercel-index#461] [Q&A] [answered]
安装完成后报404错误
https://s3.bmp.ovh/imgs/2022/02/c4acc26045505a73.png[/img]

**Comment by spencerwooo** [comment, marked as answer]
看一下 Vercel function logs？

**Reply by Genius-liu** [reply]
<img src = 'https://s3.bmp.ovh/imgs/2022/02/5f863863ebb4dd52.png' />
账号是E5管理员账号

**Comment by Genius-liu** [comment]
已经解决了，重新添加了数据库，重新部署后完成。
